.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
```

Comment:
If .idea is in the main .gitignore then this file doesn't need to be committed at all